protocol/v1/qbft/instance/msgcont/inmem/inmem_test.go

```diff
@@ -53,7 +53,7 @@ func TestFindPartialChangeRound(t *testing.T) {
 				protocoltesting.SignMsg(t, secretKeys, uids[1:2], generateConsensusMsg(0)),
 			},
 			false,
-			100000,
+			0,
```

Comment:
why we need to change here?

Reply:
@nivBlox I made `PartialChangeRoundQuorum`return `0` instead of `100000` if not found

Reply:
let's make sure it's not affecting the qbft process

Reply:
i mean, if 0 instead of 1000 is ok where we using this func

Reply:
@nivBlox yes, it's used only in `uponChangeRoundPartialQuorum` and `0` is better than `100000` there because of check that round returned from `uponChangeRoundPartialQuorum` must be higher than the round in state.